osidb/tests/factories.py

```diff
@@ -36,6 +36,7 @@ class Meta:
     cve_id = factory.sequence(lambda n: f"CVE-2020-000{n}")
     type = factory.Faker("random_element", elements=list(FlawType))
     created_dt = factory.Faker("date_time", tzinfo=UTC)
+    unembargo_dt = factory.Faker("date_time", tzinfo=UTC)
```

Comment:
This way the `FlawFactory` will always create a Flaw with `unembargo_dt` even when the Flaw it self is not embargoed, I would suggest to use `factory.Maybe` to decide based on the embargoed attribute also by default it should return some future date from now so the Flaw produced by factory can be correctly validated:

```python
unembargo_dt = factory.Maybe(
        "embargoed",
        yes_declaration=factory.Faker("future_datetime", tzinfo=UTC),
        no_declaration=None,
    )
```

Reply:
Makes sense, I will edit it.